catalog/core/catalog-core-camelcomponent/src/main/java/ddf/camel/component/catalog/content/ContentProducerDataAccessObject.java

```diff
@@ -192,25 +189,15 @@ public void createContentItem(FileSystemPersistenceProvider fileIdMap, ContentEn
 
     public void processHeaders(Map<String, Object> headers, StorageRequest storageRequest,
             File ingestedFile) {
-        String attributeOverrideHeaders = (String) headers.get(Constants.ATTRIBUTE_OVERRIDES_KEY);
-        if (StringUtils.isNotEmpty(attributeOverrideHeaders)) {
+        HashMap<String, Serializable> attributeOverrideHeaders = new HashMap<>((Map) headers.get(
```

Comment:
I'll check on that NPE.

But yeah as far as I know, since Java's `Map` interface doesn't implement `Serializable`, I won't be able to add it to the `StorageRequest` unless I cast it to a `Seriablizable`, which I can, if we're okay with that 👍